modules/fedimint-ln-server/src/lib.rs

```diff
@@ -296,7 +310,32 @@ impl ServerModule for Lightning {
         dbtx: &mut ModuleDatabaseTransaction<'b, ModuleInstanceId>,
         consensus_items: Vec<(PeerId, LightningConsensusItem)>,
     ) {
-        for (peer, decryption_share) in consensus_items.into_iter() {
+        // Separate round consensus items from decryption shares
+        let UnzipLightningConsensusItem {
+            decryption_share: decryption_shares,
+            round_consensus,
+        } = consensus_items.into_iter().unzip_lightning_consensus_item();
+
+        // FIXME: also warn on less than 1/3, that should never happen
+        // Make sure we have enough contributions to continue
+        if round_consensus.is_empty() {
+            panic!("No proposals were submitted this round");
+        }
+
+        let clock_time_proposals = round_consensus
+            .iter()
+            .map(|(_, rc)| rc.clock_time)
+            .collect();
```

Comment:
Before throwing away the peer id, do we make sure that each peer only contributed one item? Otherwise malicious ones could take over. Needs to be fixed in wallet too.